Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM node:18.8.0-alpine3.16 as base
```

Comment:
🟠 This Dockerfile in root directory is obsolete, but the one in `docker/` is not up to date

Reply:
root Dockerfile removed. docker/  Dockerfile updated